core/render/vi/__init__.py

```diff
@@ -100,6 +100,8 @@ def dumpConfig(adminTree):
                         tmp = v.copy()
                         tmp["name"] = str(tmp["name"])
                         adminConfig[key]["views"].append(tmp)
+        if "indexes" in dir(app) and key in adminConfig:
+            adminConfig[key]["indexes"] = app.indexes
```

Comment:
This would increase performance in several ways.
```suggestion
        if key in adminConfig and (indexes := getattr(app, "indexes", None)):
            adminConfig[key]["indexes"] = indexes
```